### PR TITLE
Add F# to dotnet cli 1.1.0

### PR DESCRIPTION
**Pull request by KevinRansom**
See also:   https://github.com/dotnet/cli/pull/7024

### CODE REVIEW

**Comment by KevinRansom** [comment]
@dotnet-bot test OSX10.12 x64 Release Build please

**Comment by KevinRansom** [comment]
@dotnet-bot test Linux x64 Release Build  please

**Comment by KevinRansom** [comment]
@dotnet-bot test Linux x64 Release Build please

**Comment by KevinRansom** [comment]
@dotnet-bot test this please

**Comment by nguerrera** [comment]
Fill out template and tag Matt

File: build/Microsoft.DotNet.Cli.Compile.targets [hunk 1]
```diff
@@ -194,7 +201,8 @@
                 JITPath="$(LibCLRJitPath)"
                 CrossgenPath="$(CrossgenPath)"
                 ReadyToRun="True"
-                PlatformAssemblyPaths="@(PlatformAssemblies);
+                PlatformAssemblyPaths="%(CrossgenTargets.RootDir)%(CrossgenTargets.Directory);
```
Comment (livarcocc) [inline build/Microsoft.DotNet.Cli.Compile.targets]: Why do we need this?

**Comment by KevinRansom** [comment]
Tagging @MattGertz for approval pending CI tests.

Customer scenario: VS ships with the V1.1 sdk and dotnet CLI without this change netstandard libraries and apps will not be editable in VS.
Workarounds, if any: None.
Risk:: Low
Performance impact : None only enabled for .fsproj
Is this a regression from a previous update?:, dotnet core support hasn't been provided in the past.
Root cause analysis : Not yet implemented
How was this bug found?: : Not yet implemented

**Pull request merged**